components/rich_text_editor/extensions/emoji/EmojiComponent.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <node-view-wrapper class="d-d-inline-block">
+    <dt-emoji
+      size="200"
```

Comment:
The emoji is quite small, if you look in slack for example, it seems it's quite typical to render the emoji at a size that is actually larger than the text. Maybe experiment with size 300 and 400.

![Screenshot 2023-07-21 at 11 49 34 AM](https://github.com/dialpad/dialtone-vue/assets/64808812/bffe649c-d17a-4b8c-a9a0-b685cca08767)

Reply:
yea i thought 300 was looking too big but now you mentioned slack , its a good idea to change it to that size.